File: Csharp/Creational/AbstractFactory/Example/Show/ConcreteFactory2.cs
namespace Show
{
    public class ConcreteFactory2 : IAbstractFactory
    {
        public IProductA CreateProductA()
        {
            return new ConcreteProductA2();
        }

        public IProductB CreateProductB()
        {
            return new ConcreteProductB2();
        }
    }
}

File: JavaScript/Creational/AbstractFactory/Example1/ConcreteFactory2.js
const AbstractFactory = require('./AbstractFactory');
const ProductA2 = require('./ProductA2');
const ProductB2 = require('./ProductB2');

class ConcreteFactory2 extends AbstractFactory {
    createProductA() {
        return new ProductA2();
    }
    createProductB() {
        return new ProductB2();
    }
}
module.exports = ConcreteFactory2;